### DISCUSSION TITLE
I cant publish page

### DISCUSSION THREAD

**Discussion by julik83** [community/community#30382] [Other Feature Feedback, Questions, & Ideas]
Hi there, is there anything new with publishing repository/site from github. I just cant do that, I do not have this option in my "pages", (:
Can I publish more than I site?, 
Thx in advance for any help.

**Comment by airtower-luna** [comment]
> Hi there, is there anything new with publishing repository/site from github. I just cant do that, I do not have this option in my "pages", (:

This sounds a bit confused on what exactly you're trying to do. If you want to create a new Pages site, you need to create a new repository first, and then enable Pages in the repository settings. If you want something else, please describe in more detail so people can help. :slightly_smiling_face: 

> Can I publish more than I site?

Yes.

**Comment by ichenglin** [comment]
There is an **[official tutorial](https://pages.github.com/)** about **Github pages**,
which includes a **step-by-step guide** on how to get your first **Github pages set up**.

Tutorial link: **[https://pages.github.com/](https://pages.github.com/)**

![image](https://user-images.githubusercontent.com/41904540/185932056-b50e832f-4d82-4fd9-9da1-7bae60bad1b4.png)

**Reply by queenofcorgis** [reply]
Thanks for sharing this resource @ichenglin 🗒️

**Comment by jfullstackdev** [comment]
seems the user now knows how to do it correctly, 

in one of his/her repos, we can see this and the site is already hosting when you click that

![image](https://user-images.githubusercontent.com/47092464/186659375-b6ea719d-612a-4b51-8789-214a8b430c2c.png)
